### PR TITLE
Google Workspace Sync for Microsoft Outlook 4.3.53.0

### PR DESCRIPTION
**Pull request by dw2546**
Adding as new. Silent installations fail so I removed silent modes and switches to force interactive mode.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/62787)

### CODE REVIEW

File: manifests/f/FastStone/Viewer/7.6/FastStone.Viewer.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+PackageVersion: "7.6"
+PackageLocale: en-US
+Publisher: FastStone Corporation
+# PublisherUrl: 
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+# Author: 
+PackageName: FastStone Image Viewer
+PackageUrl: https://www.faststone.org/FSViewerDetail.htm
+License: Copyright (c) 2020 FastStone Soft
```
Comment (Trenly) [inline manifests/f/FastStone/Viewer/7.6/FastStone.Viewer.locale.en-US.yaml:14]: A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate

**Comment by dw2546** [comment]
It works as expected here. Silent install not supported so the only mode in the YAML is --interactive. OutputL

C:\Users\dw254\github\winget-pkgs\manifests\g\Google\WorkspaceSync>winget install --manifest 4.3.53.0
Found Google Workspace Sync for Microsoft Outlook (GWSMO) [Google.WorkspaceSync] Version 4.3.53.0
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://dl.google.com/dl/google-apps-sync/x64/enterprise_gsync.msi
  ██████████████████████████████  61.3 MB / 61.3 MB
Successfully verified installer hash
Starting package install...
Successfully installed

File: manifests/g/Google/WorkspaceSync/4.3.53.0/Google.WorkspaceSync.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.7-2-4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Google.WorkspaceSync
```
Comment (Trenly) [inline manifests/g/Google/WorkspaceSync/4.3.53.0/Google.WorkspaceSync.installer.yaml:4]: Package has dependency on Microsoft Outlook
```
Action ended 18:31:51: InstallFinalize. Return value 3.
No supported version of Microsoft Outlook® found. Supported versions are Microsoft Outlook® 2003 SP3 (version 11.0.8169 or greater), Microsoft Outlook® 2007 SP2 (version 12.0.6504.5000 or greater), Microsoft Outlook® 2010, Microsoft Outlook® 2013, and Microsoft Outlook® 2016.
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by Trenly** [comment]
> It works as expected here. Silent install not supported so the only mode in the YAML is --interactive. Output:

Only apps which support silent install can be added to the Winget repository at the moment

**Comment by dw2546** [comment]
> 

Ok, let me see if I can get it to install silently first. If it does I will update the YAML for dependencies.

File: manifests/g/Google/WorkspaceSync/4.3.53.0/Google.WorkspaceSync.installer.yaml (hunk 2 above)
Comment (dw2546) [reply manifests/g/Google/WorkspaceSync/4.3.53.0/Google.WorkspaceSync.installer.yaml:4]: Is it possible to list this as a dependency somehow? I'm still researching this. I was able to get --silent to work.

**Review by ItzLevvie — APPROVED** [review]

**Comment by dw2546** [comment]
this was supposed to be a separate submission for google workspace. why does it keep getting mixed in with faststone image viewer?

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
> this was supposed to be a separate submission for google workspace. why does it keep getting mixed in with faststone image viewer?

Use https://github.com/microsoft/winget-create/releases/latest for manifest submission. That'll make things easier.

This is happening because your current branch (which is based on your fork's master branch) is in a dirty state due to outdated commits.

Click "Fetch and merge" on https://github.com/dw2546/winget-pkgs/tree/dw2546/winget-pkgs:
image

**Comment by zachcarp** [comment]
Once the other errors are cleaned up - if the validation error occurs I can (Or the next person on call can) manually verify installation is successful and over-ride.

**Comment by ItzLevvie** [comment]
Close with reason: Outdated merge conflicted PR;

**Pull request closed without merging**